doc/project/development_stages.md

```diff
@@ -95,23 +95,23 @@ The verification stages can be applied to simulation-based DV and formal propert
 
 The second verification stage is **Under Test**.
-In this stage, the verification environment is available but not tests in the testplan are completed.
+In this stage, the verification environment is available but not all tests in the testplan are completed and the coverpoints are not implemented.
 Once all of the items in the testplan are implemented, it exits this stage.
 
 The third verification stage is **Testing Complete**.
-In this phase, no changes are expected on the testplan, no changes expected on the testbench, and no new tests are expected except to complete full coverage of the design.
+In this phase, no changes are expected on the verification plan, no changes expected on the testbench, and no new tests are expected except to close coverage on the design.
 Once all coverage metrics have been met, waivers checked, the verification moves into its final stage: **Verification Complete**.
 
 **Stages for simulation-based DV**:
 
 | **Stage** | **Name** | **Definition** |
 | --- | --- | --- |
-| V0 | Initial Work | Testbench being developed, not functional; testplan being written; decided which methodology to use (sim-based DV, FPV, or both). |
-| V1 | Under Test | <ul> <li> Documentation: <ul> <li> [DV plan]({{< relref "doc/ug/dv_methodology#documentation" >}}) available, [testplan]({{< relref "doc/ug/dv_methodology#documentation" >}}) completed and reviewed </ul> <li> Testbench: <ul> <li> DUT instantiated with major interfaces hooked up <li> All available interface assertion monitors hooked up <li> X / unknown checks on DUT outputs added <li> Skeleton environment created with UVCs <li> TLM connections made from interface monitors to the scoreboard </ul> <li> Tests (written and passing): <ul> <li> Sanity test accessing basic functionality <li> CSR / mem test suite </ul> <li> Regressions: Sanity and nightly regression set up</ul> |
+| V0 | Initial Work | Testbench being developed, not functional; verification plan being written; decided which methodology to use (sim-based DV, FPV, or both). |
+| V1 | Under Test | <ul> <li> Documentation: <ul> <li> [DV plan]({{< relref "doc/ug/dv_methodology#documentation" >}}) available <li> [testplan]({{< relref "doc/ug/dv_methodology#documentation" >}}) completed and reviewed <li> [functional coverage plan]({{< relref "doc/ug/dv_methodology#documentation" >}}) completed and reviewed </ul> <li> Testbench: <ul> <li> DUT instantiated with major interfaces hooked up <li> All available interface assertion monitors hooked up <li> X / unknown checks on DUT outputs added <li> Skeleton environment created with UVCs <li> TLM connections made from interface monitors to the scoreboard </ul> <li> Tests (written and passing): <ul> <li> Sanity test accessing basic functionality <li> CSR / mem test suite </ul> <li> Regressions: Sanity and nightly regression set up</ul> |
```

Comment:
I think these links should probably go to `doc/ug/dv_methodology#verification-plan` (but please check - I'm not sure whether I guessed the hyphenation right).

Reply:
hi @rswarbrick 
this is as mentioned the first step - @sriyerg is updating the tooling in parallel I believe.
I didn't update the links yet as we need to decide if we merge the current testplan.hjson with coverage points and call it verification_test_plan.hjson
or if we simply keep the two separate and keep testplan.hjson and add coverage_plan.hjson.